upgrade-prechecks/preupgrade-healthcheck.sh

```diff
@@ -900,6 +900,24 @@ print_subsection
 verify_vlan
 }
 
+function verify_pid_limit(){
+    print info "Verify PID Limit"
+    rm -f ${TEMP_MMHEALTH_FILE} >> /dev/null
+    while read -r proc; do echo $proc >> ${TEMP_MMHEALTH_FILE}; done <<< "$(oc get nodes --no-headers |awk '{print $1}')"
+    while IFS= read -r line
+    do
+        pidvalue=$(oc debug nodes/$line -- chroot /host grep podPidsLimit /etc/kubernetes/kubelet.conf)
+        count=$(echo $pidvalue | grep -c "12228")
+        if [ "${count}" -ne 1 ]; then
+            print error "PID LIMIT for node $line : $(echo $pidvalue|awk '{ print $2}'|tr -d ,)"
+            print error "PID limit should be 12228"
+        else
+            print info "PID limit is okay for the node : $line"
```

Comment:
We do not print for each node if pidslimit is okay. For all nodes if pidslimit is 12228 or more we just give single green check that pidslimit on all nodes is as expected.

Reply:
done.